types/sdk.js

```diff
@@ -3,7 +3,7 @@
 /* Client */
 export type ClientRequest = {
   uri: string;
-  method: MethodType;
+  method: string;
```

Comment:
I get that, but can you investigate why? Because until now it was working.

Hint: it was implicitly defined within "flow" context. I guess in the latest versions this changed?